app.js

```diff
@@ -1,23 +1,27 @@
+require('dotenv').config()
 const express = require('express')
-const app = express();
 const mongoose = require('mongoose')
-
+const app = express()
+const blogRun = require('./routes/blogy')
```

Comment:
`blogRun` is not a good name

Reply:
Changed to Blog, I hope this name is abit fair